app/controllers/restaurants_controller.rb

```diff
@@ -1,13 +1,48 @@
 class RestaurantsController < ApplicationController
-  def index
-  end
+  	
+  	def index
+  		@restaurants = Restaurant.all
+  	end
+
+  	def show
+  		@restaurant = Restaurant.find(params[:id])
+  	end
+
+  	def new
+  		@restaurant = Restaurant.new
+  	end
+
+  	def create
+  		Restaurant.create(params.require(:restaurant)
+  			.permit(:name, :website, :created_at, :updated_at, 
```

Comment:
:created_at and :updated_at never need to be part of permitted params. Those attributes are set automatically by Rails, they're not part of user input.